.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
since we use yarn, why not using yarn here?

Reply:
No. https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem=

Reply:
This is exclatly my point, yarn exsists <img width="718" alt="Capture d’écran 2022-03-28 à 11 42 43" src="https://user-images.githubusercontent.com/1107936/160371549-87820f82-c803-427c-853c-82224ba0f8a0.png"> https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

Reply:
yaml value is "npm" 😅